user-storage-jpa/README.md

```diff
@@ -21,45 +21,27 @@ runtime. Before using this example, you should probably read the User Storage SP
 System Requirements
 -------------------
 
-You need to have <span>Keycloak</span> running.
-
-All you need to build this project is Java 8.0 (Java SDK 1.8) or later and Maven 3.3.3 or later.
-
+All you need to build this project is Java 11 (Java SDK 11) or later and Maven 3.3.3 or later.
 
 Build and Deploy the Quickstart
 -------------------------------
 
-You must first deploy the datasource it uses.
-Start up the <span>Keycloak</span> server.  Then in the directory of this example type the following maven command:
-
-   ````
-   mvn -Padd-datasource install
-   ````
+You must first configure the datasource it uses. 
+For that, copy the [conf/quarkus.properties](conf/quarkus.properties) to the `conf` directory of the server distribution.
+In this file, you have a named datasource using the same name as the [src/resources/META-INF/persistence.xml](persistence unit) from the quickstart. By using the same name,
+you make sure the persistence unit will be using the correct datasource.
 
-**Note**: If the server runs on different port than `10090`, you have to specify it by setting a
-maven property `keycloak.management.port`.
+To build the provider, run the following maven command:
 
    ````
-   mvn -Padd-datasource install -Dkeycloak.management.port=9990
+   mvn clean install
    ````
 
-You only need to execute this maven command once.  If you execute this again, then you will get an error message that the datasource
-already exists.
-
-If you open the pom.xml file you'll see that the add-datasource profile creates an XA datasource using the built
-in H2 database that comes with the server.  An XA datasource is required because you cannot use two non-xa datasources
-in the same transaction.  The <span>Keycloak</span> database is non-xa.
+To install the provider, copy the [target/user-storage-jpa-example.jar] JAR file to the `providers` directory of the server distribution.
```

Comment:
Probably you wanted to put the `target/user-storage-jpa-example.jar` in `` instead of `[]`?